Document how to reimburse cloud and add oxipng nox job (#716)
2i2c-org/team-compass: administration/reimburse.md

Ramp should scan the e-mail and automatically attach the receipt to the proper expense.
See [the Ramp receipt detection documentation](https://support.ramp.com/hc/en-us/articles/360042588454-Submitting-Receipts-Memos) for more details.

(reimburse:cloud)=
## Cloud reimbursements

> We must follow a few extra steps to reimburse our cloud bill.
> This is because cloud costs tend to be above the limit that requires extra documentation.
> Note that **CS&S requires an invoice, not a receipt** for cloud costs.
> AKA, we must have a listing of our "incurred charges for the month", not just a receipt of payment.

Author: ```suggestion
We must follow a few extra steps to reimburse our cloud bills.
CS&S requires **invoices** (not a receipts) for cloud costs.
```

We document how to find each of these for our cloud providers below.

### Google Cloud Platform (GCP)

> GCP requires that we use their cloud UI to find invoices for previous months.

Author: ```suggestion
Use the Google Cloud Console UI to find invoices for previous months.
```
